<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowGraspRRTDemo</class>
 <widget class="QMainWindow" name="MainWindowGraspRRTDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraspRrt Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <zorder>groupBox</zorder>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>430</y>
         <width>231</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Display options</string>
       </property>
       <widget class="QCheckBox" name="checkBoxColModel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Collision model</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxShowRRT">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>RRT</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxShowSolution">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Solution</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxShowSolutionOpti">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Postprocessed Solution</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxGrasps">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>20</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Grasps</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>540</y>
         <width>331</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Execute</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Position on Solution</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderPos">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>211</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>50</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="numDigits">
         <number>5</number>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonSolution">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Solution</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonSolutionOpti">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Postprocessed Sol.</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonClose">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>20</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>close hand</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonOpen">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>60</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>open hand</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBoxStart">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>331</width>
         <height>401</height>
        </rect>
       </property>
       <property name="title">
        <string>Setup</string>
       </property>
       <widget class="QPushButton" name="pushButtonLoad">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>81</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Scene</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxGoal">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Traget Object</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Conf.</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxStart">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>60</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxColChecking">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>340</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.040000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxCSpaceSampling">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>370</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.080000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>350</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Collision Checking</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>370</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CSpace Paths</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>330</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Sampling</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Robot Node Set</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxRNS">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>120</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>311</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>Collision Detection Setup</string>
        </property>
        <widget class="QComboBox" name="comboBoxColModelRobotStatic">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>50</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Robot B (e.g. body)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Robot A (e.g arm)</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxColModelRobot">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Environment Set</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxColModelEnv">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>80</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>End Effector</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxEEF">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>150</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>310</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Min Grasp Score</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMinGraspScore">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>310</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.001000000000000</double>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonGraspPose">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>370</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>test approach</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>430</y>
         <width>81</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Planning</string>
       </property>
       <widget class="QPushButton" name="pushButtonPlan">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Plan</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderPos</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>751</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
